system/security/shared/caching.cpp

```diff
@@ -579,6 +580,7 @@ bool CPermissionsCache::queryPermsManagedFileScope(ISecUser& sec_user, const cha
     if (m_managedFileScopesMap.empty())
     {
         *accessFlags = queryDefaultPermission(sec_user);
+        WARNLOG("Filescope managed scopes empty for %s, applying default permissions %s(%d), took %dms", sec_user.getName(), getSecAccessFlagName(*accessFlags), *accessFlags,  msTick()-start);
```

Comment:
Would this be better traced in the if() above where the m_managedFileScopesMap is populated?  if (count) else ...

Reply:
The if block above depends on the existence of the security manager. It is possible that one is not present.

Reply:
Added a check for no security manager in which case default permissions are being applied.